provider/pkg/gen/overlays.go

```diff
@@ -356,7 +571,7 @@ var kubeClientSettings = pschema.ComplexTypeSpec{
 		Description: "Options for tuning the Kubernetes client used by a Provider.",
 		Properties: map[string]pschema.PropertySpec{
 			"burst": {
-				Description: "Maximum burst for throttle. Default value is 10.",
+				Description: "Maximum burst for throttle. Default value is 120.",
```

Comment:
Are we just updating the description to reflect the true default value, or are we also bumbing up the QPS and burst?

Reply:
I am simply updating the description to match reality (see https://github.com/pulumi/pulumi-kubernetes/pull/2381)